scipy/linalg/tests/test_decomp.py

```diff
@@ -913,144 +903,6 @@ def test_deprecation_results(self):
         assert_allclose(v_dep, v)
 
 
-class TestLU:
-    def setup_method(self):
-        self.a = array([[1, 2, 3], [1, 2, 3], [2, 5, 6]])
-        self.ca = array([[1, 2, 3], [1, 2, 3], [2, 5j, 6]])
```

Comment:
Could we not move these tests into a separate module in this PR? I'm happy to trust you that the tests didn't change and were just copy-and-pasted, but it's still extra review effort to verify that, especially if something were to go wrong afterwards. We could do this as a separate prep PR or afterwards?

Reply:
True. The reason is that those are too simple tests. And very trivial to fit in the memory to test any array spillage etc.

That's why I replaced with more known examples further down the file. Tests are always nice to have but they should be a bit trying and not just perfectly singular arrays with integer entries. I can readd them anyways.